### PR TITLE
Initial redux integration

### PR DESCRIPTION
**Pull request by gopalprathi**
Added redux and saga in application.

### CODE REVIEW

File: autozone-poc/src/Redux/middlewares/saga.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import {take, takeEvery,call, put, select, all, fork} from 'redux-saga/effects';
+import {getproducts, getproductdetail} from '../../constants';
+import {setProducts, setProductDetail} from '../actions';
+
+const jsonUrl = 'data/products.json';
+
+function* fetchProductsSaga(){
+    console.log('fetchProductsSaga');
```
Comment (gokulwebdeveloper) [inline autozone-poc/src/Redux/middlewares/saga.js]: Remove this console.log

File: autozone-poc/src/utlis/footer/Footer.js [hunk 2]
```diff
@@ -9,11 +9,11 @@ const Footer = () => {
           <div className="row">
             <div className="col-12 col-md-6">
               <div className="single_widget_area d-flex mb-30">
-                <div className="footer-logo mr-50">
+                {/* <div className="footer-logo mr-50">
```
Comment (gokulwebdeveloper) [inline autozone-poc/src/utlis/footer/Footer.js:12]: Why you comment this line

**Review by gokulwebdeveloper — CHANGES_REQUESTED** [review]

File: autozone-poc/src/utlis/footer/Footer.js (hunk 2 above)
Comment (gopalprathi) [reply autozone-poc/src/utlis/footer/Footer.js:12]: because logo2 img file was showing 404 error.

**Review by gopalprathi — COMMENTED** [review]
@gokulwebdeveloper  Done changes.

**Pull request merged**